src/registration/scripts/directives.js

```diff
@@ -13,6 +14,17 @@
                 link: function (scope, element, attrs) {
                     scope.data = {};
 
+                    // Bind on-success attribute to success callback
+                    // scope: { onRegister: '&' }
+                    if (angular.isDefined(attrs.onRegister)) {
```

Comment:
Is there a reason why these are not passed in the scope like the `smsVerify` below? That seems a simpler option as it avoids having to do the `$parse`

```
scope: {
    onRegister: '&'
},
```

Does it break something else if the scope is totally isolated?